VISION.md

```diff
@@ -0,0 +1,17 @@
+# Material UI’s Philosophy
+
+We are on a mission to provide an elegant implementation of the Material Design Specification.
+But we believe we can do more.
+
+We want Material-UI to become whatever is generally useful for application development in the spirit of the Material Design spec.
+This means Material Design with a general use UI library of components that are needed by many. This general use doesn't imply any other design methodology.
+It also means we will have components or combinations that are simply not addressed in the spec.
+
+We also focus on providing all the low-level tools needed to build a rich user-interface with React.
+Once you solve the Material design specification (that put the bar very high),
+you should be able to take advantage of it for you own business custom style.
+We want to see companies succeeding making Material UI matches their brand. We don't want them to feel that their UI looks like another Google product.
+
```

Comment:
"We don't want them to feel that their UI looks like another Google product."

Pretty sure that's Google's intent behind Material Design! 😆